src/content/docs/en/pages/agreements/sla/sla.mdx

```diff
@@ -20,9 +20,9 @@ Service Availability will be expressed percentage wise as the total sum of Succe
 
 Azion Edge Application
 
-The availability of Edge Application, Edge Cache, Tiered Cache, Load Balancer, Image Processor, Application Accelarator, Edge Functions, Edge Firewall, Network Layer Protection and Web Application Firewall (WAF) services is to be calculated according to the same rules and their respective features correctly set up on the Azion Real-Time Manager for the Test Object.
+The availability of Edge Application, Edge Cache, Tiered Cache, Load Balancer, Image Processor, Application Accelarator, Edge Functions, Edge Firewall, Network Layer Protection and Web Application Firewall (WAF) services is to be calculated according to the same rules and their respective features correctly set up on the Azion Azion Console for the Test Object.
 
-The Monitoring Service will execute a HTTP GET operation for the Test Object and through the corresponding Azion domain. Test Objects must be delivered with HTTP headers allowing cache and TTL (Time-to-Live, i.e. the time the object will remain cached in Azion servers) equal to or longer than 2 hours and size between 1KB and 10KB. The feature to be tested must be correctly set up on the Azion Real-Time Manager for the Test Object. The interval between requests will be 5 minutes at the least and 30 minutes at the most. In return, the Monitoring Service should expect a successful HTTP code (class 2xx status code) or "Not Modified" code (304 status code). A Service Failure will be deemed to have occurred when 2 consecutive test attempts have failed in all 5 regions at the same time, except for the situations provided for in the Service Guarantee Coverage Exceptions section. Otherwise, the event will be logged as a Successful Delivery.
+The Monitoring Service will execute a HTTP GET operation for the Test Object and through the corresponding Azion domain. Test Objects must be delivered with HTTP headers allowing cache and TTL (Time-to-Live, i.e. the time the object will remain cached in Azion servers) equal to or longer than 2 hours and size between 1KB and 10KB. The feature to be tested must be correctly set up on the Azion Azion Console for the Test Object. The interval between requests will be 5 minutes at the least and 30 minutes at the most. In return, the Monitoring Service should expect a successful HTTP code (class 2xx status code) or "Not Modified" code (304 status code). A Service Failure will be deemed to have occurred when 2 consecutive test attempts have failed in all 5 regions at the same time, except for the situations provided for in the Service Guarantee Coverage Exceptions section. Otherwise, the event will be logged as a Successful Delivery.
```

Comment:
```suggestion
The Monitoring Service will execute a HTTP GET operation for the Test Object and through the corresponding Azion domain. Test Objects must be delivered with HTTP headers allowing cache and TTL (Time-to-Live, i.e. the time the object will remain cached in Azion servers) equal to or longer than 2 hours and size between 1KB and 10KB. The feature to be tested must be correctly set up on the Azion Console for the Test Object. The interval between requests will be 5 minutes at the least and 30 minutes at the most. In return, the Monitoring Service should expect a successful HTTP code (class 2xx status code) or "Not Modified" code (304 status code). A Service Failure will be deemed to have occurred when 2 consecutive test attempts have failed in all 5 regions at the same time, except for the situations provided for in the Service Guarantee Coverage Exceptions section. Otherwise, the event will be logged as a Successful Delivery.
```